decidim-proposals/config/locales/ca.yml

```diff
@@ -43,4 +43,8 @@ ca:
           send: Enviar
           title: Nova proposta
         proposal:
-          view_proposal: Veure proposta
\ No newline at end of file
+          view_proposal: Veure proposta
```

Comment:
You don't need to translate strings to all locales, Crowdin will take care of that for you. You only need to take care of the english locales, crowdin will send a PR.